src/components/Pages/QuranApps/AppsPage.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Within React 17 and newer versions come with the new JSX transforms and you don't need to import `React` anymore.

You can update the other files in this PR as well.

Reply:
I tried to remove this but it `yarn lint:` through an error and then doesn't let me commit hence can't remove the import.

Reply:
Salaam @Shemsedin,

If you add `react/jsx-uses-react: 0` right after line 47 in `.eslintrc.yml` you won't get an error anymore.

Reply:
I tried this but it is the same. Maybe if you tray on your end what works (on this branch) and let me know and I am happy to change that part.